tiledb/api/c_api/CMakeLists.txt

```diff
@@ -86,4 +86,7 @@ add_subdirectory(object)
 add_subdirectory(query)
 
 # `vfs`: depends on `context`
-add_subdirectory(vfs)
\ No newline at end of file
+add_subdirectory(vfs)
+
+# `data_order`: no dependencies
+add_subdirectory(data_order)
```

Comment:
Nit: Move this up so that we're maintaining alphabetical order

Reply:
done